azure-pipelines/templates/test.yml

```diff
@@ -17,45 +17,23 @@ jobs:
 
           - template: /azure-pipelines/templates/build-apps.yml@self
 
-          # Run tests Node 18
-          - task: NodeTool@0
-            inputs:
-                versionSpec: 18.x
-            displayName: 'Install Node 18'
-            retryCountOnTaskFailure: 10
-          - task: AzureCLI@2
-            displayName: 'Run tests Node 18'
-            continueOnError: true
-            inputs:
-                azureSubscription: 'nodejs-e2e-tests'
-                scriptType: 'bash'
-                scriptLocation: 'inlineScript'
-                inlineScript: 'npm test'
+          - template: /azure-pipelines/templates/test-node-version.yml@self
+            parameters:
+                nodeVersion: 18.x
 
           # Run tests for old config (just one Node.js version is enough)
-          - task: AzureCLI@2
-            displayName: 'Run tests old config'
-            continueOnError: true
-            inputs:
-                azureSubscription: 'nodejs-e2e-tests'
-                scriptType: 'bash'
-                scriptLocation: 'inlineScript'
-                inlineScript: 'npm run testOldConfig'
-
-          # Run tests Node 20
-          - task: NodeTool@0
-            inputs:
-                versionSpec: 20.x
-            displayName: 'Install Node 20'
-            retryCountOnTaskFailure: 10
-          - task: AzureCLI@2
-            displayName: 'Run tests Node 20'
-            continueOnError: true
-            inputs:
-                azureSubscription: 'nodejs-e2e-tests'
-                scriptType: 'bash'
-                scriptLocation: 'inlineScript'
-                inlineScript: 'npm test'
+          - template: /azure-pipelines/templates/test-node-version.yml@self
```

Comment:
Could we move this block to the end of the file or the start of the tests (~line 20)? There would be a predictable structure from lines 20-36 if we moved this block to the start or the end.